### ISSUE TITLE
Dependency on jsincss unlisted

### ISSUE THREAD

**Issue by brandonkal** [tomhodgins/deqaf#2]
This package uses `https://unpkg.com/jsincss/index.vanilla.js`, but does not specify it as a dependency or a peer dependency

Ref: https://bundlephobia.com/result?p=deqaf

**Comment by tomhodgins** [comment]
Hi @brandonkal I'd love to list this if there's a best practice way of doing this - since it's an ES module consuming an ES module it's not something that shows up under the 'dependencies' tab on npm's website either. What's the right way to go about listing this as a dependency?

**Comment by brandonkal** [comment]
Yes. I am not sure exactly what the correct approach here would be.
From reading the spec, it looks like this would work: using the url as the package name and the github location as the [version](https://docs.npmjs.com/files/package.json#github-urls).
```json
{"dependencies": {
  "https://unpkg.com/jsincss/index.vanilla.js": "tomhodgins/jsincss"
  }
}
```
Though that looks very odd! It seems package.json wasn't designed with external url dependencies in mind...

This really isn't a huge issue though as if you were looking to build an offline app, you probably would use your qaffeine library as a build step.
